devops/helmfiles/values.yaml.gotmpl

```diff
@@ -4,7 +4,7 @@ environment: {{ requiredEnv "ENVIRONMENT" | quote }}
 image:
   tag: {{ requiredEnv "IMAGE_TAG" | quote }}
 customChainspecUrl: {{ .Values | get "nodeCustomChainspecUrl" "" }}
-chain: {{ .Values | get "nodeChain" "rococo" }}
+chain: {{ .Values | get "nodeChain" "mangata-rococo-local" }}
```

Comment:
i think this should be just `mangata-rococo` ?
`rococo` previously it was the public testnet

`mangata-rococo-local` is a dev net
`mangata-rococo` is the public testnet